plugins/Files/js/components/file.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react'
+
+const File = ({filename, filesize, onDownloadClick, onDeleteClick}) => (
+	<li>
+		<div className="filename">
+			<i className="fa fa-file"></i>
+			<div className="name">{filename}</div>
+		</div>
+		<div className="file-info">
+			<span className="filesize">{Math.floor(filesize/1000000)} MB</span>
```

Comment:
we'll want something more sophisticated here, analogous to [how siac does it](https://github.com/NebulousLabs/Sia/blob/master/siac/parse.go#L16)